_includes/markdown/React.md

```diff
@@ -0,0 +1,130 @@
+The purpose of this document is to help guide you through working with a potential project using the JavaScript library, [React](https://reactjs.org/). Outside of The official React documentation, you should find here personalized recommendations for using the library based on the types of projects we typically  see at 10up. We have broken these recommendations out into common elements we tend to interact with often in client work. If something you’re looking for isn’t represented you can either submit a pull request to this repository or refer to the [official React documentation](https://reactjs.org/).
+
+## Figuring Out If React Is Right For Your Project
+
+When deciding whether React is the right tool for your project, it might help to ask the following questions:
+
+* Will this project involve building a user interface that requires statefulness? In other words, will your user interface need to...react...in real time to user actions or changeable data coming from an API?
+* Does your project require greater flexibility than a full-fledged MVC or MV(star) framework would allow? For example: does your application need to exist within a larger site context, rather than forming the basis for the entire site?
+* Does your project need enough reusable components to make it worth using React? React is a fairly heavy library compared with the similar Vue.
+* Is performance a primary concern? Will your project need to render large amounts of data in real time while remaining performant for the browser?
```

Comment:
Are we providing an answer to this question? If the answer is yes are we gonna recommend React or not?

Reply:
Maybe make that more clear? I see that in the Vue JS you explicitly say this, but I don't see this on the React section.

Reply:
Nice catch! Added